app/(docs)/object-mount/getting-started/object-mount-fusion/page.md

```diff
@@ -0,0 +1,379 @@
+
+weight: 0
+---
+
+## Overview
+{% callout type="warning"  %}
+Object Mount Fusion is currently in `BETA`. Your [feedback](https://forum.storj.io/) and [bug reports](TODO) will help us improve it.
+{% /callout %}
+
+Object Mount Fusion is way to upgrade high-performance attached storage solutions like Amazon Elastic File System (EFS) with the throughput of object storage. 
+It is a **cheaper** and **faster** solution compared to using EFS alone.
+
+Object Mount Fusion takes an attached storage filesystem and an initially empty object storage bucket/directory and exposes a single interface for both. 
+Object Mount Fusion will migrate files between object storage and local filesystem depending on where they are best for performance/cost.
+
+**Note:**
+This feature is available only to [Professional and Enterprise Tier](TODO https://cuno.io/pricing/) customers. 
+If you have any questions, please feel free to [contact us](TODO https://cuno.io/contact-us/).
+
+## How it works
+
+Object Mount Fusion combines both into a virtual mount or a FUSE mount. The files on object storage are represented as hidden links from the host filesystem to the object store. 
+Unlike other solutions, the object store is a first-class high throughput tier, rather than a slow archival tier. 
+Object Mount Fusion automatically migrates files between the two according to application behaviour on such files. 
+New files may be written in either tier, depending on predicted and observed access properties. 
+Object Mount Fusion supports multiple users simultaneously accessing files on multiple nodes by sharing the attached storage and mount location.
+
+More information can be found in the full explanation of :ref:`Object Mount Fusion < TODO user-guide-Object Mount-fusion>`.
+
+## Setting up Object Mount Fusion
+
+Object Mount Fusion is expected to be used within the same high-speed LAN as your object storage. For example, if you use AWS S3, Object Mount Fusion should only be set up on EC2 nodes within the same 
+[AWS Region](https://aws.amazon.com/about-aws/global-infrastructure/regions_az/) as the bucket to accessed. If you are using an S3-compatible on-premises object storage solution, 
+then Object Mount Fusion should be set up on a computer on the same high-speed local network.
+
+If you're already set up and using an attached storage system, such as EFS, you may skip ahead to {TODO ref}`Object Mount-fusion-guide-mount`.
+
+### Set up an empty bucket or directory on object storage
+
+For use as the object storage half of a running Object Mount Fusion mount, you will need a location on object storage that is empty. 
+This will be used as the location that the Fusion filesystem will migrate data to when it's more suitably stored on object storage. 
+This can either be an entirely empty bucket, or an empty directory on an existing bucket. This location must not be modified by anything but Object Mount Fusion filesystems.
+
+{% tabs %}
+{% tab label="Storj" %}
+    TODO
```

Comment:
@dg-io Could you find someone to complete this and the following Storj sections, please?